src/test/java/org/skriptlang/skript/test/tests/syntaxes/EvtGrowTest.java

```diff
@@ -48,9 +49,18 @@ public void setBlocks() {
 	@Test
 	public void testGrow() {
 		if (canRun) {
-			for (int i = 0; i < 10; i++) {
+			int maxIterations = 100;
+			int iterations = 0;
+			while (((Ageable) plant.getBlockData()).getAge() != ((Ageable) plant.getBlockData()).getMaximumAge()) {
```

Comment:
Could we not set its age to its maximum age here?

Reply:
I don't believe setting the age triggers the grow event sadly.